kedro-datasets/tests/api/test_api_dataset.py

```diff
@@ -153,12 +173,20 @@ def test_stream(self, requests_mock):
         chunks = list(response.iter_content(chunk_size=2, decode_unicode=True))
         assert chunks == ["I ", "am", " b", "ei", "ng", " s", "tr", "ea", "me", "d."]
 
+    def test_proxy(self, requests_mock):
     def test_proxy(self, requests_mock):
```

Comment:
Spurious line causing a `SyntaxError`

```suggestion
    def test_proxy(self, requests_mock):
```